components/common/accordianItemHeaderComponent/accordianItemHeaderComponent.tsx

```diff
@@ -0,0 +1,54 @@
+import { random_rgba } from "../../../utils/util-common/common";
+import Image from "next/image";
+import { Avatar, Stack } from "rsuite";
```

Comment:
We could have used the Oxygen UI for the sample apps. Let's create an issue if we cannot update the implementation with the current effort.

Reply:
It was discussed that oxygen UI is not necessary for this sample.